externs/lcevc.js

```diff
@@ -0,0 +1,93 @@
+/**
+* @fileoverview Externs for LcevcDil
+* compiler.
+*
+* @externs
+*/
+'use strict';
+
+var libDPIModule = {};
+var LcevcDil = {};
+
+/**
+* LCEVC DIL constructor
+* @constructor
+*/
+LcevcDil.LcevcDIL = class {
+  /**
+     *  @param {HTMLVideoElement} media
+     *  @param {HTMLCanvasElement} canvas
+     *  @param {Object=} dilConfig
+     */
+  constructor(media, canvas, dilConfig) {
+    /** @type {number} */
+    this.aspectRatio;
+
+    /** @type {Boolean} */
```

Comment:
This should be `boolean` (lowercase).  In Closure, a `boolean` (lowercase) is a plain old bool (such as `true`), but a `Boolean` (capital) is an object such as `new Boolean(true)`.

Reply:
This is rectified.